src/Attachment.tsx

```diff
@@ -175,7 +175,8 @@ export const AttachmentView = (props: {
                             <tr key={'item' + i}>
                                 <td>
                                     { item.image && <Media src={ item.image.url } onLoad={ props.onImageLoad } /> }
-                                    <span>{ item.title }</span>
+                                    <span className="title">{ item.title }</span>
+                                    <span className="subtitle">{ item.subtitle}</span>
```

Comment:
Can you wrap with renderIfNonempty - see [here](https://github.com/Microsoft/BotFramework-WebChat/blob/35a79987a1e36d5465d7aa9a6349434e72e321ce/src/Attachment.tsx#L211) for an example